src/routines.spec.ts

```diff
@@ -40,6 +45,7 @@ const actionMarbles2 = ' -a----e----a---';
 const errorMarbles = '   ------e';
 const errorSub1 = '      ^-----!';
 const errorSub2 = '      ------^--------';
+const singleActionMarble = '  -f';
```

Comment:
I think we could just reuse `a` here.

Reply:
It is just because the actionCreator accepts a specific form of string "[string] string" so I can't create an action that triggers type: 'alpha'.

Reply:
Ah ok, then this is completely fine